scripts/common.Makefile

```diff
@@ -165,7 +165,7 @@ pip-freeze: ## dumps current environ and base.txt [diagnostics]
 
 .PHONY: _check_python_version _check_venv_active
 
-_check_python_version:
+_check_python_versio%:
```

Comment:
what is this `%` doing here?

Reply:
This allows to override that recipe in the director without a warning. This is to makefile what * is for glob